apps/passport-server/src/database/models.ts

```diff
@@ -330,3 +330,10 @@ export interface AnonMessageWithDetails extends AnonMessage {
   topic_name: string;
   reactions: string[];
 }
+
+export interface RateLimit {
+  type: "CHECK_EMAIL_TOKEN" | "REQUEST_EMAIL_TOKEN";
```

Comment:
nit: may be worth extracting the type of the `type` field into its own standalone type. maybe a `const enum`